ansible/roles/nginx-rails/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
   client_max_body_size: 4G
   ssl_force_redirect: true
   default_server: true
+  keepalive_timeout: 10
+  extra_nginx_config: {}
```

Comment:
is this easier to do as a big text block vs a hash?  Don't some nginx options (like logging one example) have complicated and sometimes even multiline configurations?

Reply:
can you do a big text block in yaml?

Reply:
Yes and apparantly there is an entire website dedicated to figuring out the syntax 😆 
https://yaml-multiline.info/